src/file.jl

```diff
@@ -42,9 +42,6 @@ function h5open(
         flag = swmr ? API.H5F_ACC_TRUNC | API.H5F_ACC_SWMR_WRITE : API.H5F_ACC_TRUNC
         fid = API.h5f_create(filename, flag, fcpl, fapl)
     else
-        ishdf5(filename) || error(
-            "unable to determine if $filename is accessible in the HDF5 format (file may not exist)"
-        )
```

Comment:
It would be great if you could add another condition to the `if` statement rather than deleting this. If you cannot just comment it out, and I can look into it later.